### PR TITLE
FN-10715 Completely rewrite frontend logic, add support for COMPANY_BNPL (PragmaGO)

### PR DESCRIPTION
**Pull request by akozubskicr**
Completely rewrite frontend logic, add support for a new payment method - pay later for companies (COMPANY_BNPL).

### CODE REVIEW

**Review by pkowalczykk — APPROVED** [review]

File: includes/assets/js/comfino.js [hunk 1]
```diff
@@ -1,267 +1,24 @@
...
-        });
+        let script = document.createElement('script');
 
-        document.getElementById('modal-repr-example').querySelector('button.comfino-modal-exit').addEventListener('click', function (event) {
-            event.preventDefault();
-            document.getElementById('modal-repr-example').classList.remove('open');
-        });
+        script.onload = () => ComfinoFrontendRenderer.init(Comfino.options);
+        script.src = frontendScriptURL;
+        script.async = true;
```
Comment (pkowalczykk) [inline includes/assets/js/comfino.js:18]: 👍🏻

**Review by pkowalczykk — APPROVED** [review]
dziwne te pliki z tłumaczeniami

File: readme.txt [hunk 2]
```diff
@@ -4,13 +4,16 @@ Donate link: https://comfino.pl/
 Tags: comfino, woocommerce, gateway, payment, bank
 WC tested up to: 7.8.2
```
Comment (mchylekcr) [inline readme.txt:5]:
```suggestion
WC tested up to: 8.0.3
```

File: readme.txt [hunk 3]
```diff
@@ -4,13 +4,16 @@ Donate link: https://comfino.pl/
...
 WC requires at least: 3.0
-Stable tag: 3.0.0
+Stable tag: 3.1.0
 Tested up to: 6.2.2
```
Comment (mchylekcr) [inline readme.txt:8]:
```suggestion
Tested up to: 6.3.1
```

**Pull request merged**